app/assets/javascripts/show-hide-content.js

```diff
@@ -0,0 +1,172 @@
+;(function (global) {
```

Comment:
No, I wasn't familiar with rails asset pipeline, and was making sure it worked before I put that in.  Managed to miss this, I've put in a pull request for this now.

https://github.com/alphagov/email-alert-frontend/pull/34